versioned_docs/version-1.25/self-hosted/helm-configuration.mdx

```diff
@@ -203,10 +203,10 @@ The build service. It's used in combination with `okteto build` to build contain
 - `serviceAccount.name`: Buildkit service account name. Defaults to `okteto-buildkit`.
 - `serviceAccount.annotations`: Annotations for the buildkit service account.
 - `serviceAccount.labels`: Labels for the buildkit service account.
-- `persistence.enabled`: Configures a persistence volume for buildkit. False by default.
+- `persistence.enabled`: Configures a persistence volume for buildkit. Enabled by default.
 - `persistence.class`: The storage class of the persistence volume attached to every buildkit pod.
 - `persistence.size`: The size of the persistence volume attached to every buildkit pod. Defaults to `750Gi`.
-- `persistence.cache`: The size (in Mi) of the buildkit cache to store image caches. It should be around 30Gi smaller than `storage.size`. Defaults to 500Gi.
```

Comment:
@maroshii should you also remove mentions to:
```
buildkit:
  persistence:
    enabled: true
```
in our cloud provider installations guides and `config.yaml` files like `src/content/self-hosted/eks/config.yaml`?
I think it's cleaner if we remove them since it's already the default value